.gitignore/test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
- [ ]  Good job so far but you  still need some changes for your project to look awesome :wink:  , You should ignore the `test.md` and `node modules` before you push on GitHub, They can't be present in your repository, Please  ignore them in `.gitignore`  :grin: :grin:   
This is how it should look like :arrow_down: 

![gitignore](https://user-images.githubusercontent.com/88894525/191379955-23a3aa35-f866-49d3-aa15-6eeb6b7192ce.png)

Reply:
Hello @Alfredbis29, I already made the change. Can you let me know if it's ok now? I don't want to fail my last coding review 🥹

Reply:
@Alfredbis29 and also can you let me know how you check if they are really being ignored?

Reply:
hi You can just create a file called gitignore then you put inside those two files which are test.md and nodes_modules then you push on your branch

Reply:
Hi, I did it just as you showed me, can you tell me if I'm correct? 

<img width="1013" alt="Screen Shot 2022-09-20 at 19 49 54" src="https://user-images.githubusercontent.com/90674847/191384480-83e0464b-15e3-4c41-afb4-c74cd7a83180.png">